packages/app/src/components/time-series-chart/components/tooltip/tooltip-series-list-items.tsx

```diff
@@ -124,20 +129,21 @@ function TooltipListItem({
   children,
   icon,
   label,
+  ariaLabel,
```

Comment:
```suggestion
  label,
  ariaLabel = label,
```

defeats the need of multiple `ariaLabel ?? label`